buddyboss/buddyboss-platform
# PROD-928 - Mentioning a member on a blog post is not working on subsite

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1293

### How to test the changes in this Pull Request:

1. Create a WP multisite.
2. Create users in a subsite.
3. Mention other sub-sites users in a current subsite. and It should work.

### Other information:
